Ascend.java

```diff
@@ -2,6 +2,14 @@
 // and prints them in increasing order.
 public class Ascend {

```

Comment:
OVERALL:
1. This is an OK submission, I have 3 issues regarding logic in Ascend.java, NumWords.java and TimeFormat.java please review my notes, and please contact me if you have any question. 
2. I want you to do another review on conditions... In TimeFormat.java the issues of when we want to use condition is evident, it is ok for that time of the course. After you review the material, I want you to rewrite TimeFormat.java and send it to my email, **tal.danai@post.runi.ac.il**. 
3. Regarding styling please go over the styling guidelines.